snet-ui/AirdropRegistrationMini/index.tsx

```diff
@@ -22,12 +24,24 @@ const DateFormatter = new Intl.DateTimeFormat("en-GB", {
   // timeZone: "UTC",
   timeZoneName: "short",
 });
-export default function AirdropRegistrationMini({ startDate, totalTokens, tokenName }: AirdropRegistrationMiniProps) {
-  const formattedDate = useMemo(() => DateFormatter.format(startDate), [startDate]);
+export default function AirdropRegistrationMini({
+  startDate,
+  totalTokens,
+  tokenName,
+  totalAirdropWindows,
+  currentAirdropWindow,
+}: AirdropRegistrationMiniProps) {
+  const formattedDate = useMemo(() => DateFormatter.format(startDate), [
+    startDate,
+  ]);
   return (
-    <GradientBox $background="bgGradientHighlight" sx={{ px: 4, pt: 4, pb: 5, borderRadius: 2 }}>
+    <GradientBox
+      $background="bgGradientHighlight"
+      sx={{ px: 4, pt: 4, pb: 5, borderRadius: 2 }}
+    >
       <Typography color="text.secondary" variant="h4" align="center" mb={6}>
-        Airdrop registration window starts {formattedDate}
+        Airdrop registration window {currentAirdropWindow}/{totalAirdropWindows}{" "}
```

Comment:
The text displayed in the AirdropRegistrationMini should be based on the current time and window. 
It should not be always a registration window as it is being displayed at multiple timelines whenever the user account is not connected.

Reply:
Fixed in the latest commit